packages/rum-core/src/browser/performanceCollection.ts

```diff
@@ -199,12 +225,15 @@ export function retrieveInitialDocumentResourceTiming(
     let timing: RumPerformanceResourceTiming
 
     const forcedAttributes = {
-      entryType: 'resource' as const,
+      entryType: RumPerformanceEntryType.RESOURCE as const,
```

Comment:
<!-- Nitpicks are small, trivial, but necessary changes. Distinguishing nitpick comments significantly helps direct the reader's attention to comments requiring more involvement.  -->
**🥜 nitpick:** ‏the `as const` is probably unnecessary now

Reply:
Without it, it seems like typescript complains  
```
Types of property 'entryType' are incompatible.
    Type 'RumPerformanceEntryType' is not assignable to type 'RumPerformanceEntryType.RESOURCE'
```

Reply:
Ah right nevermind!